### ISSUE TITLE
Time for introducing a develop branch?

### ISSUE THREAD

**Issue by leastbad** [hopsoft/stimulus_reflex#84]
As the project continues to pick up steam, I suspect that it will become increasingly difficult to manage the flurry of feature, release and hotfix branches without some kind of agreed-upon structure.

I was re-reading the classic [A successful Git branching model](https://nvie.com/posts/a-successful-git-branching-model/) and would like to propose that we move towards adopting this model, starting with a develop branch. Feature branches are created from develop and merged back into it.

One of the first motivations I have is around our evolving documentation efforts. Right now, GitBook is connected and making commits directly on master. Those commits are live immediately, which causes chronology issues for features that haven't been publicly released yet.

Possibly making the question rhetorical... the best idea I had was to create a 2nd GitBook project and point it to the develop branch. We could continue to do our modifications using the GitBook UI and the results would be a preview of what is soon to be released (hopefully) to master. Of course, once develop is merged into master, GitBook should now reflect up-to-date documentation for the now-current new release.

Anyhow, @hopsoft @andrewmcodes what do you folks feel about this?

**Comment by leastbad** [comment]
Closing because silence 😶
